--- nginx.spec.org	2015-10-26 17:37:32.000000000 +0900
+++ nginx.spec	2015-10-26 17:39:14.000000000 +0900
@@ -139,6 +139,8 @@
         --with-debug \
         %{?with_http2:--with-http_v2_module} \
         --with-cc-opt="%{optflags} $(pcre-config --cflags)" \
+        --add-module=/usr/local/src/ngx_mruby \
+        --add-module=/usr/local/src/ngx_devel_kit \
         $*
 make %{?_smp_mflags}
 %{__mv} %{_builddir}/%{name}-%{version}/objs/nginx \
@@ -180,6 +182,8 @@
         --with-ipv6 \
         %{?with_http2:--with-http_v2_module} \
         --with-cc-opt="%{optflags} $(pcre-config --cflags)" \
+        --add-module=/usr/local/src/ngx_mruby \
+        --add-module=/usr/local/src/ngx_devel_kit \
         $*
 make %{?_smp_mflags}
 
