--- nginx.spec.orig     2016-03-16 13:11:12.754043330 +0000
+++ nginx.spec  2016-03-16 13:11:44.234690754 +0000
@@ -91,6 +91,9 @@
         --with-mail_ssl_module \
         --with-file-aio \
         --with-ipv6 \
+        --without-stream_access_module \
+        --add-module=/usr/local/src/ngx_mruby \
+        --add-module=/usr/local/src/ngx_devel_kit \
         %{?with_http2:--with-http_v2_module}")

 Summary: High performance web server
